react-components/select/step6.md

```diff
@@ -0,0 +1,55 @@
+Typeahead variants support adding new options, based on the user entered input. This is supported in two parts, first by allowing a user to select their custom input, and second by allowing this custom input to be added to the overall list of options for permanence.
+
+1) **Add the `isCreateable` flag to the `Select` component.**
+
+<pre class="file" data-target="clipboard">
+isCreatable
+</pre>
+
+The component should look as below.
```

Comment:
I think you can remove this part as they only added one line (unless you think its necessary for them to check this at this point)